### PR TITLE
feat(Pony Town) new presence

### LINKED ISSUE #5416: Pony | pony.town
### Discussed in https://github.com/PreMiD/Presences/discussions/4514

<div type='discussions-op-text'>

<sup>Originally posted by **GhostNyah** November  8, 2020</sup>
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?**  Ghost.#3405

**What's the name of the service?**  https://pony.town

**What should the Presence display?**  display the elapsed game time, and the game photo

**If possible, please provide a logo for the service (512x512 minimum)**  
image

</div>

### PR DESCRIPTION
**Pull request by lavylavenderr**
## Description 

A presence for a simple, but growing site. Closing [this](https://github.com/PreMiD/Presences/issues/5416) service request.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image
image
image

</details>

### CODE REVIEW

File: websites/P/Pony Town/presence.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const presence = new Presence({
+		clientId: "995451252588695663",
+	}),
+	websiteLoadTimestamp = Math.floor(Date.now() / 1000);
```
Comment (kyrie25) [inline websites/P/Pony Town/presence.ts:4]:
```suggestion
	browsingTimestamp = Math.floor(Date.now() / 1000);
```

File: websites/P/Pony Town/presence.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	if (document.location.pathname === "/") presenceData.details = "In-Game";
+	else if (document.location.pathname.includes("help"))
+		presenceData.details = "Viewing the help page.";
+	else if (document.location.pathname.includes("about"))
+		presenceData.details = "Viewing the about page.";
+	else if (document.location.pathname.includes("character"))
+		presenceData.details = "Customizing their character.";
+	else if (document.location.pathname.includes("account"))
+		presenceData.details = "Editing their account settings.";
+	presenceData.startTimestamp = websiteLoadTimestamp;
```
Comment (kyrie25) [inline websites/P/Pony Town/presence.ts:19]: Merge this onto the `presenceData` constant above and change it to `browsingTimestamp`

File: .gitignore [hunk 3]
```diff
@@ -17,4 +17,4 @@ package-lock.json
 **/.history
 
 # Exception for extension recommendations
-!**/.vscode/extensions.json
\ No newline at end of file
+!**/.vscode/extensions.json
```
Comment (kyrie25) [inline .gitignore]: Might wanna revert this change

**Review by kyrie25 — CHANGES_REQUESTED** [review]
Run `yarn format` to Prettify the files.

File: websites/P/Pony Town/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+	"author": {
+		"name": "Lavenderleaff",
+		"id": "205984221859151873"
+	},
+	"contributors": [
+		{
+			"name": "joerkig",
+			"id": "205984221859151873"
+		}
+	],
```
Comment (EncryptedDev) [inline websites/P/Pony Town/dist/metadata.json:12]: A contributor must have contributed to at least 20% of the code base. This is unlikely because it is a new PR

File: websites/P/Pony Town/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+	"altnames": [
+		"Pony"
+	],
+	"description": {
+		"en": "A game of ponies building a town."
+	},
+	"url": "pony.town",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/aZXi6K1.png",
+	"thumbnail": "https://i.imgur.com/aZXi6K1.png",
```
Comment (EncryptedDev) [inline websites/P/Pony Town/dist/metadata.json:23]: Logo and thumbnail cannot be the same

File: websites/P/Pony Town/dist/metadata.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+	"contributors": [
+		{
+			"name": "joerkig",
+			"id": "205984221859151873"
+		}
+	],
+	"service": "Pony Town",
+	"altnames": [
+		"Pony"
+	],
```
Comment (EncryptedDev) [inline websites/P/Pony Town/dist/metadata.json:16]: I don't think the service also goes by this name

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Comment by EncryptedDev** [comment]
You should also support pages with dynamic features 😕

**Comment by Lavenderleaff** [comment]
> You should also support pages with dynamic features 😕

Unsure if the info was available, also make all requested changes.
